package.json

```diff
@@ -11,6 +11,8 @@
     "start": "cd packages/core && NODE_ENV=production node .",
     "cli": "logto",
     "alteration": "logto db alt",
+    "//": "# `changeset version` won't run version lifecycle scripts, see https://github.com/changesets/changesets/issues/860",
+    "ci:version": "changeset version && pnpm -r version",
```

Comment:
nit: Why don't we put this command directly in the `release.yml`?

Reply:
it's a universal script

Reply:
so we'd better to duplicate the original behavior instead of depending on workflow